share/namespace.go

```diff
@@ -106,6 +107,7 @@ func (n Namespace) Validate() error {
 }
 
 // ValidateForData checks if the Namespace is of real/useful data.
+// TODO: it's not clear what "real/useful" data means.
```

Comment:
@Wondertan @vgonkivs can you please help me understand what "real/useful" data means? Do we need `ValidateForData` or is `ValidateForBlob` sufficient?

Reply:
If we need to maintain this function, it may need to be updated in this PR.

Reply:
Not parity data